mtp_cashbook/apps/disbursements/forms.py

```diff
@@ -177,12 +177,13 @@ class SendingMethodForm(DisbursementForm):
 
 
 class RecipientContactForm(DisbursementForm):
-    recipient_name = forms.CharField(label='Their name')
+    recipient_name = forms.CharField(label='Their first name')
+    recipient_last_name = forms.CharField(label='Last name')
     address_line1 = forms.CharField(label='Their address')
     address_line2 = forms.CharField(required=False)
-    city = forms.CharField(required=False)
-    postcode = forms.CharField(label='Their postcode')
-    email = forms.EmailField(label='Their email address (if provided)', required=False)
+    city = forms.CharField(label='Town or city', required=False)
+    postcode = forms.CharField(label='Postcode')
+    email = forms.EmailField(label='Their email address (optional)', help_text='This is to confirm the payment request has been sent', required=False)
```

Comment:
this line is too long so you need to break it up somewhere. there's a limit of 120 characters or something